docs/hpc/advanced/profiling/nsight_systems.md

```diff
@@ -0,0 +1,68 @@
+# Nsight Systems
+
+[NVIDIA Nsight Systems](https://developer.nvidia.com/nsight-systems) is a system-wide performance analysis tool designed to visualize an application’s algorithms, identify the largest opportunities to optimize, and tune to scale efficiently across any quantity or size of CPUs and GPUs, from large servers to our smallest systems-on-a-chip (SoCs).
+
+## Profile the code using nsight systems
+
+You can profile the code interactively or via a batch job.
+
+1. The first step is to load the appropriate modules.
+
+```bash
+module load tools/prod
+
+# Load the appropriate cuda module. We recommend using new versions as older versions are not supported on our systems for profiling.
+module load CUDA/12.2.0
+
+# Load a Qt module. This is because Nsight Systems uses Qt for its GUI. 
+module load Qt5/5.15.5-GCCcore-11.3.0
+```
+
+2. To profile the code either interactively on the login node or via a batch job, you can use the following command.
+
+```bash
+nsys profile --trace cuda,osrt,nvtx --force-overwrite true --output <output_file> <your_app> <your_app_parameters>
+```
+
+where,
+`output_file` is the name of the file in which nsight systems will save the profiling data.
+`your_app` is the name of the executable of your application.
+`your_app_parameters` are the parameters that you pass to your application.
+
+Please note that `force-overwrite` will overwrite the output file if it already exists. If you want to keep the old data, you can remove this option.
+
+For example, to profile a simple cuda Jacobi program, you can use the following command.
```

Comment:
cuda needs capitalising

Reply:
Corrected in 3d807fc507e7bdcfdb2dd978f9094d45d6b680c3 and 1b89b8fdac7d75afc158fe5655619fa0fd48357c